libpod/container_internal.go

```diff
@@ -898,6 +909,7 @@ func (c *Container) generateSpec() (*spec.Spec, error) {
 
 	g.SetHostname(c.Hostname())
 	g.AddProcessEnv("HOSTNAME", g.Spec().Hostname)
+	g.AddProcessEnv("CONTAINER", "libpod")
```

Comment:
This should be lower-case.

Reply:
Good point, fixed

Reply:
I think this should be done by buildah and podman not by the library.
I would rather see the fact that buildah created me or podman rather then libpod.

Reply:
Alright, fair enough.